srcs/Client.cpp

```diff
@@ -9,7 +9,7 @@ Client::Client(){};
  * @param new_fd socket fd
  * @param _mode mode for USER, CHANNEL
  */
-Client::Client(int new_fd): _clientFd(new_fd), _serverName("🐾TYCHUNEN SERVER🐾"), _nickname(""), _mode("+Ziw"), _operatorPassword("TYCHUNEN"),_isRegistered(0), _welcomeSent(0), _isOperator(false) {}
+Client::Client(int new_fd): _clientFd(new_fd), _serverName("🐾TYCHUNEN SERVER🐾"), _nickname(""), _mode("+Ziw"), _operatorPassword("TYCHUNEN"),_isRegistered(0), _welcomeSent(0), _isOperator(false), _maxChannels(0) {}
```

Comment:
is this param max amount of channels per cllient or just the amount of channels user has joined?

Reply:
it is badly named. just amount of channels user has joined. Maximum amount of channels that client can have is 3 :) and it is just manually written in join cmd -> clientErrorCheck function